packages/flutter_tools/lib/src/commands/start.dart

```diff
@@ -165,3 +165,10 @@ Future<int> startApp(
 
   return startedSomething ? 0 : 2;
 }
+
+String _getRelativePath(String fullPath) {
```

Comment:
Hmm, I wasn't aware. I looked at it - it'll always try and relativize the name. What I want here is a good, short display name. So, relative if it's contained in the cwd, and absolute otherwise. I renamed this to `_getDisplayPath()` to clarify -